grammar.js

```diff
@@ -18,15 +18,25 @@ module.exports = grammar({
     word: $ => $.identifier,
 
     extras: $ => [
-        $.comment,
+        $.line_comment,
+        $.block_comment,
         // https://github.com/tree-sitter/tree-sitter-javascript/blob/2c5b138ea488259dbf11a34595042eb261965259/grammar.js#L11
         /[\s\uFEFF\u2060\u200B\u00A0]/
     ],
 
+    externals: $ => [
+        $.block_comment,
+    ],
+
     rules: {
         source_file: $ => seq(repeat($.enable_directive), repeat($._declaration)),
 
-        comment: $ => seq("//", /.*/),
+        comment: $ => choice(
+            $.line_comment,
+            $.block_comment,
+        ),
+
```

Comment:
```suggestion
```
I believe we can remove the `comment` rule, as it's not longer used inside `extras`.

Reply:
I left it in since that's what the rust grammar does. I don't know why they'd do that, but my guess is it's so you can uses it as an entry point for parsing. Otherwise I don't see a reason for it either.

> https://github.com/tree-sitter/tree-sitter-rust/blob/f7fb205c424b0962de59b26b931fe484e1262b35/grammar.js#L1442

Reply:
The only change that I can notice after removing that rule and running the `tree-sitter generate` is inside the `grammar.json` file which [appears to contain the same information as the `grammar.js`](https://tree-sitter.github.io/tree-sitter/implementation#parsing-a-grammar). I think it's safe to remove it.